classes/class.ilTestCronPlugin.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+require_once __DIR__ . "/../vendor/autoload.php";
+
+use iLUB\Plugins\TestCron\Helper\DIC;
+use iLUB\Plugins\TestCron\Helper\TestCronDBAccess;
+use iLUB\Plugins\TestCron\Jobs\RunSync;
+
+/**
+ * Class ilTestCronPlugin
+ *
+ * @package
+ */
+class ilTestCronPlugin extends ilCronHookPlugin {
+
+	use DIC;
+
+	const PLUGIN_ID = 'tcron';
+	const PLUGIN_NAME = 'TestCron';
+	const TABLE_NAME = 'tcron_config';
+	const COLUMN_NAME = 'expiration';
+	const DEFAULT_EXPIRATION_VALUE = 240;
+    const EXPIRATION_THRESHOLD = 'expiration_threshold';
+    const ILPLUGIN_TABLE = 'il_plugin';
+    const LOG_DESTINATION = './Customizing/global/plugins/Services/Cron/CronHook/TestCron/app.log';
```

Comment:
Never ever log to the web dir of ILIAS. This is important. Can you not use the ILIAS logger for that of at least the ilias file system abstraction (see e.g. the logger in hub2).